data/data/bootstrap/systemd/units/crio-configure.service.template

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
Can we make sure we capture this as part of the bootstrap-gather @cgwalters [here](https://github.com/openshift/installer/blob/master/data/data/bootstrap/files/usr/local/bin/installer-gather.sh)